src/config/app.js

```diff
@@ -1,4 +1,6 @@
 // app-wide settings (no user changable settings here)
 module.exports = {
   PACKAGE_MANAGER: 'yarn',
+  // Enable logging, if enabled all terminal responses are visible in the console (useful for debugging)
+  LOGGING: true,

```

Comment:
Actually, is this something that should be turned on by default or should it only be turned on when needed?

Reply:
OK, I'll turn it off so it needs to be enabled if required. I just had it enabled for the branch, so we can test it.
But I think we need to add an info to the docs. So it's clear for new collaborators that we have it. I'll add an info to our docs issue #210 as a reminder.